src/main/java/seedu/address/logic/commands/meeting/LeaveMeetingViewCommand.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.commands.meeting;
```

Comment:
I think one `leave` command would be enough. But we can keep this for now.

Reply:
Alright.